src/organisms/contentCard/styles.tsx

```diff
@@ -0,0 +1,86 @@
+import { Styled } from '../../theme';
+import { ContentCardProps, ContentType } from './types';
+
+export const Card = Styled.div<{ size: ContentCardProps['size'] }>`
+    border-radius: 12px;
+    box-shadow: ${({ theme }) => theme.dropShadow.REGULAR};
+    background: ${({ theme }) => theme.containerAndCardShades.SHADE_PLUS_3};
+    width: ${({ size }) => (size === 'S' ? 321 : 420)}px;
```

Comment:
@ArinCantCode why its locked to 420px ? , we can have for flexible card if we gonna do max-width
![Screen Shot 2022-06-19 at 10 04 33](https://user-images.githubusercontent.com/35912161/174469839-89f2abe0-b205-4ccc-8d98-84352b14e1fd.png)

Reply:
Question is how do we want to go about responsiveness, Michal usually creates different variations for element if they should change in different screen sizes, for example button has multiple variations including the special small button.

I imagine that those sizes there are the only ones we are going to use, therefor i do the same for other components, if there are no multiple sizes provided i assume we cap that component at specific size.

I'm aware we have some special cases sometimes, for example latest announcement card can have dynamic size based on current implementation. So question here is should that card ever scale or if it should just jump between breakpoints. 

In general i think its better to have capped width (not percentages/vw/vh units) because the behavior of elements is more predictable, if you change something you know its working if you see it, with dynamic heights you have to make sure that on the entire breakpoint range nothing wraps weirdly etc.